modules/cert-manager-certificate-api-server.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-creating-certificate.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-certificate-api-server_{context}"]
+= Creating certificates for API server
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed {cert-manager-operator} 1.13.0 or above.
+
+.Procedure
+
+. Create an issuer. For more information, see "Configuring an issuer" in "Additional Resources" section.
+
+. Create a certificate:
+
+.. Create a YAML file, for example, `certificate.yaml`, that defines the `Certificate` object:
++
+.Example `certificate.yaml` file
++
+[source, yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: <tls_cert> #<1>
+  namespace: openshift-config
+spec:
+  isCA: false
+  commonName: "<common_name>" #<2>
+  secretName: <tls_cert> #<3>
+  dnsNames:
+  - "api.<sub.domai.ns-domain-x.y.z.com>" #<4>
```

Comment:
Same here, preferably:
```
commonName: "api.<cluster_base_domain>"
```
and 
```
dnsNames:
- "api.<cluster_base_domain>"
```

User can get the value of `<cluster_base_domain>` from `oc get dns cluster -o jsonpath='{.spec.baseDomain}'` which is okay if not explicitly added as they can also get from elsewhere.